README.md

```diff
@@ -93,12 +93,23 @@ create /etc/eremetic/eremetic.yml with:
     logformat: json
 
 ## Database Backing
-Eremetic uses a BoltDB based database to store task information. The location of
-it can be set by adding
+Eremetic uses a database to store task information. The driver can be configured
+by setting the `database_driver` value to either `zk` or `boltdb`.
 
-    database: '/tmp/eremeticdb/eremetic.db'
+The location of the database can be configured by setting the `database` value.
 
-to the eremetic.yml
+### BoltDB
+The default database that will be used unless anything is configured.
+
+The default location for it is `db/eremetic.db`
+
+### ZooKeeper
+If you use `zk` as a database driver, the default location of the database is set
+to the zookeeper configured in the `master` field, but `/mesos` will be replaced
+by `/eremetic`.
+
+Note:
+`/eremetic` will always be appended to a `zk` database path.
```

Comment:
I have not looked at the rest of the yet but the most obvious way of doing it is letting the dbpath be the full zk:///-url including the hosts and any prefix you want to use. Just like the master url is written.

It's tempting to be clever by reusing the master url but you run the risk of shooting yourself in the foot when the assumptions fail. E.g on our clusters the masters are actually on `/<clustername>` so we would end up nesting eremetic data under the master tree